kSuite/kSuitePro/src/main/kotlin/com/infomaniak/core/ksuite/ksuitepro/views/components/ProOfferContent.kt

```diff
@@ -0,0 +1,222 @@
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.core.ksuite.ksuitepro.views.components
+
+import android.content.res.Configuration
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import com.infomaniak.core.compose.basics.Dimens
+import com.infomaniak.core.compose.basics.Typography
+import com.infomaniak.core.compose.margin.Margin
+import com.infomaniak.core.ksuite.data.KSuite
+import com.infomaniak.core.ksuite.ksuitepro.R
+import com.infomaniak.core.ksuite.ksuitepro.data.ProFeature
+import com.infomaniak.core.R as RCore
+
+@Composable
+fun ProOfferContent(
+    kSuite: KSuite,
+    isAdmin: Boolean,
+    onClick: () -> Unit,
+) {
+
+    val title = computeTitle(kSuite)
+    val description = computeDescription(kSuite)
+    val features = computeFeatures(kSuite)
+
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .background(colorResource(R.color.kSuiteContentBackground)),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+
+        val paddedModifier = Modifier.padding(horizontal = Margin.Large)
```

Comment:
Put this below Image, so we understand quickly why you didn't put it as the general padding